src/map/label.cpp

```diff
@@ -145,60 +145,60 @@ void map_labels::set_team(const team* team)
 }
 
 const terrain_label* map_labels::set_label(const map_location& loc,
-		const t_string& text,
-		const int creator,
-		const std::string& team_name,
-		const color_t color,
-		const bool visible_in_fog,
-		const bool visible_in_shroud,
-		const bool immutable,
-		const std::string& category,
-		const t_string& tooltip)
-{
-	terrain_label* res = nullptr;
-
-	// See if there is already a label in this location for this team.
-	// (We do not use get_label_private() here because we might need
```

Comment:
Why did you remove this comment? I don't see anything to suggest it's no longer relevant?